docs/DOCKER-MACHINE.md

```diff
@@ -0,0 +1,45 @@
+# Docker Machine
+
+Before the Docker native client we used docker-machine. Here are the steps for this original setup process.
+
+## Docker dependencies
+
+The project is managed via Docker containers. Right now we're using Docker
+version 1.9.0 with Docker Machine version 0.5.0.
+
+Commands below assume OS X and preference to install libraries via Homebrew.
```

Comment:
I think it's alright to do so.  Perhaps you want to check with @andydutch if anyone in his side is not using Docker native.